src/libsync/CMakeLists.txt

```diff
@@ -82,6 +77,12 @@ target_link_libraries(libsync PUBLIC
     csync
     Qt5::Core Qt5::Network
 )
+target_compile_definitions(libsync
+    PUBLIC -DQT_NO_CAST_TO_ASCII
```

Comment:
private

Reply:
All of them? Only this one?

Reply:
We have to remove the first one altogether, the rest should be private, in order to make the build work.

Reply:
All